bindings/python/py_src/gtars/refget/__init__.pyi

```diff
@@ -0,0 +1,109 @@
+from typing import Union, Optional, List, Type
+from enum import Enum
+from os import PathLike
+
+class AlphabetType(Enum):
+    """
+    Represents the type of alphabet for a sequence.
+    """
+    Dna2bit: int
+    Dna3bit: int
+    DnaIupac: int
+    Protein: int
+    Ascii: int
+    Unknown: int
+
+    def __str__(self) -> str: ...
+
+class SequenceMetadata:
+    """
+    Metadata for a biological sequence.
+    """
+    name: str
+    length: int
+    sha512t24u: str
+    md5: str
+    alphabet: AlphabetType
+
+    def __repr__(self) -> str: ...
+    def __str__(self) -> str: ...
+
+class SequenceRecord:
+    """
+    A record representing a biological sequence, including its metadata and optional data.
+    """
+    metadata: SequenceMetadata
+    data: Optional[bytes] # Vec<u8> maps to bytes in Python
```

Comment:
Confirming this one? Its both 1) sometimes `None` and then 2) the data can only be bytes?